server/routes/sentencingMigration/sentencingMigrationController.ts

```diff
@@ -30,8 +31,10 @@ export default class SentencingMigrationController {
 
     const decoratedMigrations = migrations.map(history => ({
       ...history,
-      applicationInsightsLink: SentencingMigrationController.applicationInsightsUrl(
-        SentencingMigrationController.alreadyMigratedApplicationInsightsQuery(history.whenStarted, history.whenEnded),
+      applicationInsightsLink: alreadyMigratedLogAnalyticsLink(
+        'Will not migrate the adjustment',
```

Comment:
this is nonsense, but couldn't find the better text in the from nomis service

Reply:
another idempotent migration that will never request an entity - so link is irrelevant